_maps/RandomRuins/SpaceRuins/chaosmarine.dmm

```diff
@@ -0,0 +1,598 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/turf/open/space/basic,
+/area/space)
+"b" = (
+/turf/closed/indestructible/cult,
+/area/ruin/space/has_grav/powered)
+"c" = (
+/obj/effect/portal/permanent/one_way{
```

Comment:
can you cheese this fight by just setting up an emitter to shoot into the one-way portal, waiting a few minutes, and then collecting your loot?

Reply:
He will teleport behind you if you aggro him out of range, so it's more of a way to cheeze him out of the arena.